apps/astarte_pairing/lib/astarte_pairing/queries.ex

```diff
@@ -175,10 +186,12 @@ defmodule Astarte.Pairing.Queries do
   def fetch_registered_devices_count(realm_name) do
     keyspace = Realm.keyspace_name(realm_name)
 
+    consistency = Consistency.domain_model(:read)
```

Comment:
In other places we are reading / writing the device table with `Consistency.device_info(:read)` and `Consistency.device_info(:write)`.
We should stick to that to keep the read and write patterns balanced.